ush/rocoto/workflow_utils.py

```diff
@@ -295,7 +295,10 @@ def get_resources(machine, cfg, task, reservation, cdump='gdas'):
         ppn = cfg['npe_node_%s' % ltask]
 
     if machine in [ 'WCOSS_DELL_P3', 'HERA', 'ORION']:
-        threads = cfg['nth_%s' % ltask]
+        if cdump in ['gfs'] and 'nth_%s_gfs' % task in cfg.keys():
+            threads = cfg['nth_%s_gfs' % ltask]
+        else:
+            threads = cfg['nth_%s' % ltask]
```

Comment:
More with the second resource specification.

Reply:
reverted